tox.ini

```diff
@@ -17,6 +18,6 @@ passenv =
     LABELBOX_TEST_API_KEY_ONPREM 
     LABELBOX_TEST_API_KEY_CUSTOM     
     LABELBOX_TEST_GRAPHQL_API_ENDPOINT     
-    DA_GCP_LABELBOX_API_KEY 
+    DA_GCP_LABELBOX_API_KEY
 
-commands = pytest {posargs}
+commands = pytest -k "not notebooks" {posargs}
```

Comment:
Curious, does this exclude the `test_notebooks.py` from the automation?

Reply:
Yes, it excludes them from the ci job. Eventually we can add it there, but need to sort out certain things like what happens when we reach the max limit of models we are allowed to have in an org.